frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -128,6 +128,29 @@ class SceneController {
       return cube;
     };
 
+    let renderedLines = [];
+
+    window.addLine = (a: Vector3, b: Vector3) => {
+      const material = new THREE.LineBasicMaterial({ color: 0x0000ff });
+
+      const points = [];
+      points.push(new THREE.Vector3(...a));
+      points.push(new THREE.Vector3(...b));
+
+      const geometry = new THREE.BufferGeometry().setFromPoints(points);
+
+      const line = new THREE.Line(geometry, material);
+      this.rootNode.add(line);
+      renderedLines.push(line);
+    };
+
+    window.removeLines = () => {
+      for (const line of renderedLines) {
+        this.rootNode.remove(line);
+      }
+      renderedLines = [];
+    };
```

Comment:
I used that code to visualize the scan lines which the oblique bucket picker casts. Might be useful in the future, too, which is why I left it here (next to the debugging visualization of buckets).

Reply:
out-of-scope for this PR, but maybe it would makes sense to put these debug helpers in an object (e.g. `window.__debug`) to not clutter the window object?

Reply:
Good point. I created https://github.com/scalableminds/webknossos/issues/5290